analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParameterValidationInYieldShouldBeWrapped.cs

```diff
@@ -144,7 +144,7 @@ class NullCoalescingAssignment
 {
     IEnumerable<int> NullableInlineArray(int? i) // FN
     {
-        _ = i ?? throw new ArgumentNullException(nameof(i));
+        i ??= throw new ArgumentNullException(nameof(i));
```

Comment:
Update: it actually doesn't compile - unlike the null coalescing operator (`??`), the null coalescing assignment (`??=`) doesn't support `throw` statements, since it would require explicit support from the language ([here](https://github.com/dotnet/csharplang/blob/main/meetings/2018/LDM-2018-07-16.md#throw-expression-on-the-right-hand-side) are some details about the decision of not supporting it - yet).